docs/Tutorials/Quick-Start.md

```diff
@@ -168,4 +175,13 @@ echo $this->Form->file('file');
 echo $this->Form->error('file');
 echo $this->Form->submit(__('Upload'));
 echo $this->Form->end();
+```
+
+Products Image View
+-------------
+
+```php
+// found in src/Template/Products/view.php
+
+<?=  $this->Image->display($product->productImages[0],'medium',['width'=>'200px','height'=>'200px']); ?> 
```

Comment:
No short tags please as the other lines don't use them either. Also there are missing spaces, please respect coding standards.